manifests/init.pp

```diff
@@ -197,6 +202,7 @@
   Pulpcore::ChecksumTypes $allowed_content_checksums = ['sha224', 'sha256', 'sha384', 'sha512'],
   String[1] $remote_user_environ_name = 'HTTP_REMOTE_USER',
   Integer[0] $worker_count = min(8, $facts['processors']['count']),
+  Boolean $use_legacy_tasking_system = true,
```

Comment:
To avoid the ole `legacy` wording, perhaps we name it `use_postgresql_tasking_system` or `use_rq_tasking_system.

Reply:
or an Enum of `['rq', 'postgresql']` or `['rq', 'new']`?

If I understand https://github.com/pulp/pulpcore/commit/0e1c6292f6601be4ea93417d688bb929b8a88212 correctly, there are two changes in that commit:
1. add a script ("entrypoint" in Python lingo) `pulpcore-worker` which should be used instead of `rq`
2. allow for two different tasking systems: old RQ and new PostgreSQL based

So switching to the old one doesn't *require* us to switch to calling `rq` directly (like this PR does right now) and I think this is actually officially unsupported by Pulpcore (or *should* be unsupported)

And if you look at the commit in `pulp_installer` (https://github.com/pulp/pulp_installer/commit/39a886965dca3cc30778329d7b1ccfbd32bb7b18) they switch to the new binary for all 3.13+ installs, unrelated to the used tasking system

Reply:
I went with `use_rq_tasking_system` for now but I'm open to other ideas.

Some thoughts: at a base level it would be nice for this module to be consistent with the setting name in pulpcore, but `USE_NEW_WORKER_TYPE` has a problem in that the postgresql tasking system won't be "new" forever, and something that is described as "new" could be perceived by users as being unstable, unsafe, etc.

I like being specific and descriptive about what technology is actually used, so `Enum['rq', 'postgresql']` is appealing for that reason, but that could be more difficult for `foreman-installer` users ("what were the options again? I need to find that in the docs somewhere") vs. a simple boolean which can be set true or false.

So I went with `use_rq_tasking_system`, added a note in the parameter documentation that this refers to the older rq workers tasking system and the alternative is the postgresql tasking system introduced in pulpcore 3.13.

When we later reach a point where `use_rq_tasking_system` is false by default, this seems particularly intuitive as an interface to allow users to opt-in to use the older tasking system instead.